android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -726,6 +726,45 @@ public void peerConnectionAddICECandidate(ReadableMap candidateMap, final int id
         callback.invoke(result);
         Log.d(TAG, "peerConnectionAddICECandidate() end");
     }
+    private WritableArray convertWebRTCStats(StatsReport[] wrtcReports) {
+        WritableArray reports = Arguments.createArray();
+        for (StatsReport wrtcReport : wrtcReports) {
+            WritableMap report = Arguments.createMap();
+            report.putString("id", wrtcReport.id);
+            report.putString("type", wrtcReport.type);
+            report.putDouble("timestamp", wrtcReport.timestamp);
+            WritableArray values = Arguments.createArray();
+            for (StatsReport.Value v : wrtcReport.values) {
+                WritableMap keyValue = Arguments.createMap();
+                keyValue.putString(v.name, v.value);
+                values.pushMap(keyValue);
+            }
+            report.putArray("values", values);
+            reports.pushMap(report);
+        }
+        return reports;
+    }
+    @ReactMethod
+    public void peerConnectionGetStats(String trackId,
+                                       int id, final Callback statsCb) {
+        PeerConnection peerConnection = mPeerConnections.get(id);
+        if (peerConnection != null) {
+            MediaStreamTrack mediaTrack = mMediaStreamTracks.get(trackId);
+            if (trackId == null || trackId.isEmpty() || mediaTrack != null) {
```

Comment:
I don't get this condition. The value of trackId was already used on the previous line to get MediaStreamTrack by trackId. Why don't you check the value of trackId before the call to get?

I know you don't like assignments in conditions but the easiest way to fix this condition I see is:
````
    MediaStreamTrack mediaTrack;
    if (trackId == null
            || trackId.isEmpty()
            || (mediaTrack = mMediaStreamTracks.get(trackId)) != null) {
````